darwin/server/default.nix

```diff
@@ -15,7 +15,7 @@
   programs.man.enable = lib.mkDefault config.srvos.server.docs.enable;
 
   # remove uninstaller, use 'nix run github:LnL7/nix-darwin#darwin-uninstaller'
-  system.includeUninstaller = lib.mkDefault false;
+  system.tools.darwin-uninstaller.enable = lib.mkDefault false;
```

Comment:
Out of curiosity, is there a reason in particular you don't want to include the uninstaller by default? Part of the reason we include it by default is so that you don't need a working Nix that can build derivations in case a user accidentally uninstalled Nix before nix-darwin

Reply:
See https://github.com/LnL7/nix-darwin/issues/804 for more context

Reply:
The `server` modules disable stuff like this, docs, etc to improve closure size and eval.